File: Berger.Data/MsfaEntity/ELearning/UserQuestionAnswer.cs
using Berger.Data.Common;
using Berger.Data.MsfaEntity.Users;
using System.Collections.Generic;

namespace Berger.Data.MsfaEntity.ELearning
{
    public class UserQuestionAnswer : AuditableEntity<int>
    {
        public int UserInfoId { get; set; }
        public UserInfo UserInfo { get; set; }
        public int QuestionSetId { get; set; }
        public QuestionSet QuestionSet { get; set; }
        public int TotalMark { get; set; }
        public int TotalCorrectAnswer { get; set; }
        public bool Passed { get; set; }
        public IList<UserQuestionAnswerCollection> QuestionAnswerCollections { get; set; }
    }
}

File: BergerMsfaApi/ClientApp/src/app/Shared/Entity/ELearning/userQuestionAnswer.ts
import { QueryObject } from '../Common/query-object';
import { UserInfo } from '../Users/userInfo';
import { QuestionSet } from './questionSet';
import { UserQuestionAnswerCollection } from './userQuestionAnswerCollection';

export class UserQuestionAnswer {
    id: number;
    // userInfoId: number;
    // userInfo: UserInfo;
    userFullName: string
    employeeId: string
    // questionSetId: number;
    // questionSet: QuestionSet;
    questionSetTitle: string;
    questionSetLevel: string;
    examDate: string;
    totalMark: number;
    passMark: number;
    userMark: number;
    // totalCorrectAnswer: number;
    // passed: boolean;
    passStatus: string;
    // status: number;
    // questionAnswerCollections: UserQuestionAnswerCollection[];

    // statusText: string;
    // passedText: string;
    // questionSetTitle: string;
    // questionSetLevel: string;
    // questionSetTotalMark: string;
    // questionSetPassMark: string;
    
    constructor(init?: Partial<UserQuestionAnswer>) {
        Object.assign(this, init);
    }

    clear() {
        // this.id = null;
        // this.userInfoId = null;
        // this.userInfo = null;
        // this.userFullName = '';
        // this.employeeId = '';
        // this.questionSetId = null;
        // this.questionSet = null;
        // this.examDate = '';
        // this.totalMark = null;
        // this.totalCorrectAnswer = null;
        // this.passed = null;
        // this.status = null;
        // this.questionAnswerCollections = [];
        // this.statusText = '';
        // this.passedText = '';
        // this.questionSetTitle = '';
        // this.questionSetLevel = '';
        // this.questionSetTotalMark = '';
        // this.questionSetPassMark = '';
    }
}

export class UserQuestionAnswerQuery extends QueryObject  {
    
    constructor(init?: Partial<UserQuestionAnswerQuery>) {
        super();
        Object.assign(this, init);
    }

    clear() {
    }
}